src/components/Home.jsx

```diff
@@ -81,14 +95,15 @@ const Home = () => {
   return (
     <>
       <div className={Styles.books}>
-        {bookElements}
+        {allBooks}
       </div>
       <AddBookForm />
     </>
   );
 };
 
 Book.propTypes = {
+  bookId: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   author: PropTypes.string.isRequired,
   category: PropTypes.string,
```

Comment:
- [ ] It will be better to separate `AddBookForm` and `allBooks` into separate components to make your clean and organized

Reply:
@omarsalem7 Thanks for reviewing my work. However, the components in my code are already separated. I don't understand what you're suggesting.